jsonschema2pojo-core/src/main/java/org/jsonschema2pojo/AbstractAnnotator.java

```diff
@@ -95,4 +95,8 @@ public GenerationConfig getGenerationConfig() {
         return generationConfig;
     }
 
+	@Override
```

Comment:
Many of these changes are using tabs instead of spaces. Could you remove all tabs from this PR and use spaces instead?